### PR TITLE
Refuse to run trusted setup if it already exists

### LINKED ISSUE #189: install.sh: Separate `npm install` from `setupAll`
# Description
At the moment in `install.sh`, we've bundled `npm ci` commands (for each microservice) with the setupAll command:
```
#!/usr/bin/env bash

# Exit script as soon as a command fails.
set -o errexit

cd zkp-utils && npm ci && \
cd ../account-utils && npm ci && \
cd ../zkp && npm ci && \
npm run setup-all && cd ../
```
As a developer, I want to be able to quickly install stuff, without having to do a zk-SNARK setup.

Can we add a separate `install-dev.sh` script (or some other name) which does not contain the setupAll script? That way if I want to quickly install stuff, I can, with less faff.

### PR DESCRIPTION
**Pull request by vishnusudarshan**
# Description

This PR is regarding updates in install.sh file for giving a choice for users to recreate any specific proof. The Github Actions for this PR will fail because of the changes in install.sh file. The PR should be merged only after the Github Actions are upgraded. 

## Related Issue

Close #189 

## How Has This Been Tested

- Tested in Command Line

## Screenshots (if appropriate)
Screenshot 2019-10-28 at 11 17 59 AM

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -1,8 +1,87 @@
...
+while true; do
+    if [ $# -ne 0 ]
+    then
+        yn=$1
+    else
+        read -p "Do you want to recreate all trusted setups? (y/n) : " yn    
+    fi
+    case $yn in
+        [Yy]* ) npm run setup-all && cd ../; break;;
+        [Nn]* ) echo -e '\033[32mList of all trusted steups :\033[m'
```
Comment (iAmMichaelConnor) [inline install.sh]: Typo: 'steups' -> 'setups'

File: install.sh [hunk 2]
```diff
@@ -1,8 +1,87 @@
...
+do
+    echo $i
+done
+yn=''
+while true; do
+    if [ $# -ne 0 ]
+    then
+        yn=$1
+    else
+        read -p "Do you want to recreate all trusted setups? (y/n) : " yn
```
Comment (iAmMichaelConnor) [inline install.sh]: recreate -> rerun

File: install.sh [hunk 3]
```diff
@@ -1,8 +1,87 @@
...
+        yn=$1
+    else
+        read -p "Do you want to recreate all trusted setups? (y/n) : " yn    
+    fi
+    case $yn in
+        [Yy]* ) npm run setup-all && cd ../; break;;
+        [Nn]* ) echo -e '\033[32mList of all trusted steups :\033[m'
+                echo -e '1.ft-mint\n2.nft-mint\n3.ft-transfer\n4.nft-transfer\n5.ft-burn\n6.nft-burn'
+                echo -e 'Type the number of each trusted setup seperated by a comma'
+                read -p '(use "1,2" to create trusted setup for ft-mint & nft-mint) : ' proofs
```
Comment (iAmMichaelConnor) [inline install.sh]: use "1,2" to create trusted setup for ft-mint & nft-mint -> E.g. type "1,2" to run the trusted setups for ft-mint & nft-mint

Comment (vishnusudarshan) [reply install.sh]: Done.

File: install.sh (hunk 2 above)
Comment (vishnusudarshan) [reply install.sh]: Done.

File: install.sh (hunk 1 above)
Comment (vishnusudarshan) [reply install.sh]: Done.

**Comment by ChaitanyaKonda** [comment]
#297 includes these codes changes. Hence closing this due to redundancy.

**Pull request closed without merging**